boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_partition_conf.dts

```diff
@@ -49,11 +49,11 @@
 };
 
 &sram0_s {
-	reg = <0x20000000 0x10000>;
+	reg = <0x20000000 0x40000>;
 };
 
 &sram0_ns {
-	reg = <0x20010000 0x60000>;
+	reg = <0x20040000 0x30000>;
```

Comment:
I think you need to also update this line accordingly:
https://github.com/zephyrproject-rtos/zephyr/blob/882d28b289d0c0d9053d6adffdb1b53a4fcc3849/boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_common.dts#L196

Reply:
done